DataFormats/GEMRecHit/interface/ME0Segment.h

```diff
@@ -22,14 +22,14 @@ class ME0Segment final : public RecSegment {
  public:
 
   /// Default constructor
-  ME0Segment() : theChi2(0.){}
+  ME0Segment() : theChi2(0.), theTimeValue(0.), theTimeUncrt(0.), theDeltaPhi(0.){}
 	
   /// Constructor
   ME0Segment(const std::vector<const ME0RecHit*>& proto_segment, const LocalPoint& origin, 
 	     const LocalVector& direction, const AlgebraicSymMatrix& errors, double chi2);
 
   ME0Segment(const std::vector<const ME0RecHit*>& proto_segment, const LocalPoint& origin, 
-	     const LocalVector& direction, const AlgebraicSymMatrix& errors, double chi2, double time, double timeErr);
+	     const LocalVector& direction, const AlgebraicSymMatrix& errors, double chi2, float time, float timeErr, float deltaPhi);
```

Comment:
The base class, "RecSegment" returns chi2 in double precision. I think it is fine for us to change it to a float here.